### PR TITLE
it is a quick update for dependencies

### PR DESCRIPTION
**Pull request by keremciu**
I will update index files but It's working with this setup

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,7 +21,7 @@ if (!global.L) {
     var L = require(leafletPath);
     global.L = L;
 
-    var scriptLength = leafletPath.split(path.sep).slice(-1)[0].length;
+    var scriptLength = leafletPath.split('/').slice(-1)[0].length;
```
Comment (jieter) [inline index.js]: I think your PR description means you are going to fix this?

Comment (keremciu) [inline index.js]: ups, I forgot it, I've updated now but tests are not working good, I'm trying to fix them but node-canvas is working nicely, it fixes my problem

**Comment by jieter** [comment]
Hmm, you're also adding a couple of unrelated changes. I've copied the version updates to my local working copy, but the test fail, no images are being generated at all. I'll have a look into it.

**Comment by jieter** [comment]
superseded by b766f3a7e2cdfb8c4c5f17239ef78061dd7f7131

File: package.json [hunk 2]
```diff
@@ -18,19 +18,19 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "dependencies": {
-    "canvas": "~1.3.0",
-    "jsdom": "~3.1.2",
-    "leaflet": "^1.0.0-rc.1",
-    "leaflet-image": "^0.2.1",
-    "request": "^2.65.0"
+    "canvas": "1.6.2",
```
Comment (keremciu) [inline package.json]: @jieter we need new versions, like these ones

**Pull request closed without merging**